templates/cisco_xr_admin_show_environment_power.textfsm

```diff
@@ -0,0 +1,8 @@
+Value POWER_MODULE (\d+\/PM\d+)
```

Comment:
```suggestion
Value PS_MODULE (\S+)
```

So the value name matches an existing show environment power (`cisco_ios_show_environment_power_all.textfsm`)

Is there any other values for status other than **OK** or **Failed**? We can change the regex around for the `Parse` matching from all the space, non whitespace to just .+? and have the regex for status be `(OK|Failed)`.